src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/ISimdVector_2.cs

```diff
@@ -0,0 +1,450 @@
+        /// <param name="vector">The vector to get the first element from.</param>
+        /// <returns>A scalar <typeparamref name="T" /> containing the value of the first element.</returns>
+        /// <exception cref="NotSupportedException">The type of the elements in the vector (<typeparamref name="T" />) is not supported.</exception>
+        static abstract T ToScalar(TSelf vector);
+
+        /// <summary>Tries to copy a <see cref="Vector{T}" /> to a given span.</summary>
+        /// <param name="vector">The vector to copy.</param>
+        /// <param name="destination">The span to which <paramref name="destination" /> is copied.</param>
+        /// <returns><c>true</c> if <paramref name="vector" /> was successfully copied to <paramref name="destination" />; otherwise, <c>false</c> if the length of <paramref name="destination" /> is less than <see cref="Count" />.</returns>
+        /// <exception cref="NotSupportedException">The type of the elements in the vector (<typeparamref name="T" />) is not supported.</exception>
+        static virtual bool TryCopyTo(TSelf vector, Span<T> destination)
+        {
+            if (!TSelf.IsSupported)
+            {
+                ThrowHelper.ThrowNotSupportedException(ExceptionResource.Arg_TypeNotSupported);
+            }
+
+            if ((uint)(destination.Length) < (uint)(TSelf.Count))
+            {
+                return false;
+            }
+
+            ref byte address = ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(destination));
+            Unsafe.WriteUnaligned(ref address, vector);
+
+            return true;
+        }
+
+        /// <summary>Creates a new <see cref="Vector128{T}" /> with the element at the specified index set to the specified value and the remaining elements set to the same value as that in the given vector.</summary>
+        /// <param name="vector">The vector to get the remaining elements from.</param>
+        /// <param name="index">The index of the element to set.</param>
+        /// <param name="value">The value to set the element to.</param>
+        /// <returns>A <see cref="Vector128{T}" /> with the value of the element at <paramref name="index" /> set to <paramref name="value" /> and the remaining elements set to the same value as that in <paramref name="vector" />.</returns>
+        /// <exception cref="ArgumentOutOfRangeException"><paramref name="index" /> was less than zero or greater than the number of elements.</exception>
+        /// <exception cref="NotSupportedException">The type of the elements in the vector (<typeparamref name="T" />) is not supported.</exception>
+        static abstract TSelf WithElement(TSelf vector, int index, T value);
+
+        // TODO: static abstract operator *(TSelf left, T right);
+        // TODO: As<TTo>()
+        // TODO: Ceiling()
+        // TODO: Dot
+        // TODO: ExtractMostSignificantBits
```

Comment:
Much like `Vector<T>`, this can't be exposed like it is on `Vector64/128/256<T>`

Instead, we'd have to expose this as returning some `VectorMask<T>`. That then complicates things quite a bit since its another piece of info required to "use" the type. If .NET had "associated types", this would be "nicer".